.github/workflows/push.yml

```
@@ -0,0 +1,70 @@
+##############################################################################
+##############################################################################
+#
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)